pkg/document/json/array.go

```diff
@@ -214,7 +214,9 @@ func (p *Array) insertAfterInternal(
 		ticket,
 	))
 
-	p.InsertAfter(prevCreatedAt, value)
+	if err = p.InsertAfter(prevCreatedAt, value); err != nil {
+		panic(err)
```

Comment:
Could you tell me why panic is called?
I think it's safer to handle errors using errors.

Reply:
Oh, are you going to phase it out?

Reply:
I thought it is okay to call panic in `json` according to https://github.com/yorkie-team/yorkie/issues/497#issuecomment-1585657549.
It would be great to change the above logic to handle errors if possible. 
I guess one of possible errors can be an error that occurs because of failure of finding a node for the given createdAt. 
```
func (a *RGATreeList) findNextBeforeExecutedAt(
	createdAt *time.Ticket,
	executedAt *time.Ticket,
) (*RGATreeListNode, error) {
	node, ok := a.nodeMapByCreatedAt[createdAt.Key()]
	if !ok {
		return nil, fmt.Errorf("fail to find the given createdAt: %s", createdAt.Key())
	}

	...
}

```

Would you mind giving me some guides or examples to safely handle errors like the above?